### PR TITLE
neovim: update to 0.8.2, adopt

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/neovim/template [hunk 1]
```diff
@@ -12,8 +12,8 @@ short_desc="Fork of Vim aiming to improve user experience, plugins and GUIs"
 maintainer="Steve Prybylski <sa.prybylx@gmail.com>"
 license="Apache-2.0, custom:Vim"
 homepage="https://neovim.io"
-distfiles="https://github.com/neovim/neovim/archive/v${version}.tar.gz"
-checksum=b4484e130aa962457189f3dee34b8481943c1e395d2d684c6f8b91598494d9ec
+distfiles="https://github.com/neovim/neovim/archive/refs/tags/v${version}.tar.gz"
```
Comment (kruceter) [inline srcpkgs/neovim/template:15]: Is there a specific need to add `refs/tags/` in `distfiles`?

Comment (tranzystorek-io) [reply srcpkgs/neovim/template:15]: Makes the link consistent with what github release page gives, and avoids potential ambiguity

Comment (paper42) [reply srcpkgs/neovim/template:15]: When /archive is used and a tag and branch of the same name exist, GitHub will not know which one we want. This bit us in the past and GitHub now gives this URL instead, so we are sometimes switching to it.
It's not anything important, but when someone switches to it, we let it go through and when someone tries to switch back to archive, we ask them to not do that.

**Review by paper42 — COMMENTED** [review]
The maintainer has not been active since 2019, would you like to adopt this package?

**Comment by tranzystorek-io** [comment]
> The maintainer has not been active since 2019, would you like to adopt this package?

Sure, let me grab the field

**Pull request merged**